lncfg/db.go

```diff
@@ -23,10 +26,13 @@ const (
 	BoltBackend                = "bolt"
 	EtcdBackend                = "etcd"
 	PostgresBackend            = "postgres"
+	SqliteBackend              = "sqlite"
 	DefaultBatchCommitInterval = 500 * time.Millisecond
 
 	defaultPostgresMaxConnections = 50
 
+	defaultSqliteDBName = "lnd.db"
```

Comment:
Deviating from `channeldb.db` to make it easier to check at a glance if it's bolt or sqlite? We could consider adding another level of a sub-dir here (`sqlite/`) which also makes it a bit easier to group the existing databases.